### ISSUE TITLE
Access Denied on MobileNet downloading

### ISSUE THREAD

**Issue by jazzisnice** [tensorflow/tensorflow#15009]
Trying to run /tensorflow/examples/image_retraining/retrain.py, getting the following error: 

```
  File "/home/burjanviktor/DeepLearning/dog_breed/retraining.py", line 1032, in main
    maybe_download_and_extract(model_info['data_url'])
  File "/home/burjanviktor/DeepLearning/dog_breed/retraining.py", line 344, in maybe_download_and_extract
    filepath, _ = urllib.request.urlretrieve(data_url, filepath, _progress)
... 
  File "/home/burjanviktor/anaconda3/lib/python3.6/urllib/request.py", line 650, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

I am trying to use MobileNet for retraining, but the following url: 

'http://download.tensorflow.org/models/mobilenet_v1_1.0_160_quantized_frozen.tgz'

gives the following error: 

```
<Error>
<Code>AccessDenied</Code>
<Message>Access denied.</Message>
<Details>Anonymous users does not have storage.objects.get access to download.tensorflow.org/models/mobilenet_v1_1.0_160_quantized_frozen.tgz.</Details>
</Error>
```

The 1.0_220 version works, but 1.0_190 does not work also.

Could you advise on this issue please?

**Comment by tatatodd** [comment]
It looks like the only quantized model we have available is this one:
http://download.tensorflow.org/models/mobilenet_v1_1.0_224_quantized_frozen.tgz

Non-quantized models are available for all other sizes (as described in `retrain.py`).  E.g. the non-quantized version of 1.0_160 is available:
http://download.tensorflow.org/models/mobilenet_v1_1.0_160_frozen.tgz

This corresponds to the following argument to `retrain.py`:
```
--architecture mobilenet_1.0_160
```

@suharshs might have more details on whether we plan to make more quantized models available.

**Comment by suharshs** [comment]
Thanks Todd. We currently only have size 224 image mobilenet quantized model available. We haven't yet prioritized getting trained quantized models for the other inputs sizes. I'll rename this issue as a feature request to help us prioritized training the other models, thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by zerodefects** [comment]
Hi @suharshs .  Are there any new quantized models available / in the works?  Perhaps some of the tiny 0.25 models ;).  Thanks!

**Comment by suharshs** [comment]
In the works :) Will keep you posted!

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by raghuraman-k** [comment]
We should have updated models available next week.
Thanks!

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by suharshs** [comment]
I have pushed changes to the retrain.py script to support these, once the next sync happens.

Commands like
```
python tensorflow/examples/image_retraining/retrain.py \
    --image_dir ~/flower_photos/   --architecture mobilenet_1.0_224_quant
```
should work for all depth multipliers and image sizes.

Please create a new issue if things don't work once github pulls in the changes to retrain.py

**Comment by tiandiao123** [comment]
Is this correct way to re-train quantized mobilenet? I mean use re-train.py?

**Comment by zerodefects** [comment]
Yes.  Retrain.py architecture switch supports the quant models, as it is described above.  Just make sure you are using a somewhat recent version of TensorFlow with a recent version of the retrain.py script.
